code/modules/mob/living/carbon/xenomorph/castes/ravager/castedatum_ravager.dm

```diff
@@ -9,7 +9,7 @@
 	wound_type = "ravager" //used to match appropriate wound overlays
 
 	// *** Melee Attacks *** //
-	melee_damage = 26
+	melee_damage = 25
```

Comment:
just keep this at 26

Reply:
also be aware the damage reduction impacts plasma generation, so if you dont want that you may increase a bit the plasma gen on hit value

Reply:
Implemented your requests from discord